patterns/404.php

```diff
@@ -7,10 +7,12 @@
 
 ?>
 
-<!-- wp:heading {"textAlign":"center","level":1,"fontSize":"x-large"} -->
-<h1 class="has-text-align-center has-x-large-font-size" id="oops-that-page-can-t-be-found"><?php echo esc_html__( 'Oops! That page can&rsquo;t be found.', 'twentytwentyfour' ); ?></h1>
+<!-- wp:heading {"textAlign":"left","level":1,"style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}}},"textColor":"contrast","fontSize":"x-large"} -->
+<h1 class="wp-block-heading has-text-align-left has-contrast-color has-text-color has-link-color has-x-large-font-size" id="page-not-found"><?php echo esc_html__( 'Page Not Found', 'twentytwentyfour' ); ?></h1>
```

Comment:
Do we need the ID? I don't know how that got there and what it's here for

Reply:
Which ID are you talking about? @MaggieCabrera

Reply:
id="page-not-found but I see that it's generated by GB to act as an anchor so this is good to go, sorry for the confusion!